Library/Homebrew/keg_relocate.rb

```diff
@@ -5,9 +5,10 @@ class Keg
   PREFIX_PLACEHOLDER = "@@HOMEBREW_PREFIX@@"
   CELLAR_PLACEHOLDER = "@@HOMEBREW_CELLAR@@"
   REPOSITORY_PLACEHOLDER = "@@HOMEBREW_REPOSITORY@@"
+  LIBRARY_PLACEHOLDER = "@@HOMEBREW_REPOSITORY@@/Library"
```

Comment:
Why is this preferable to `@@HOMEBREW_LIBRARY@@`?

Reply:
The location of `Library` is always precisely `$HOMEBREW_REPOSITORY/Library`, so it seems unnecessary to introduce a new placeholder token. Introducing a new token would make rollout of the PR trickier, as CI could not build bottles using this new token until a tagged stable release is rolled out to users who are on stable.

Reply:
> The location of `Library` is always precisely `$HOMEBREW_REPOSITORY/Library`

That's currently the case but I don't think it's a safe assumption that it always will be (or that it will be obvious that this code needs changed in that case). If there's e.g. a test verifying this: fair enough.

> Introducing a new token would make rollout of the PR trickier, as CI could not build bottles using this new token until a tagged stable release is rolled out to users who are on stable.

This seems fine to me, we could set it with an environment variable.

Reply:
I've added a test to ensure that `HOMEBREW_LIBRARY = HOMEBREW_REPOSITORY/Library`.

It's already true that for bottles that refer to files inside `@@HOMEBREW_REPOSITORY@@`, if the file layout structure of those files were to change, that bottle would no longer work. For example…
```console
$ pkg-config --variable pc_path pkg-config
/usr/local/lib/pkgconfig:/usr/local/share/pkgconfig:/usr/lib/pkgconfig:/usr/local/Homebrew/Library/Homebrew/os/mac/pkgconfig/10.15
```
If the directory `Library` in `HOMEBREW_REPOSITORY` were renamed to something else, `pkg-config` would no longer be able to find the `.pc` files provided by Homebrew.

Reply:
> I've added a test to ensure that `HOMEBREW_LIBRARY = HOMEBREW_REPOSITORY/Library`.

Thanks! Can you add a comment to both there and here explaining the assumption and that if one is edited they both need to be.

Reply:
Done!